src/core/dbus-cgroup.c

```diff
@@ -346,6 +348,39 @@ static int property_get_ip_address_access(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_pinned_bpf_program(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
```

Comment:
style nit: empty line after {

Reply:
@bluca fixed that and couple other nitty places